examples/tohoku_inversion/plot_convergence.py

```diff
@@ -0,0 +1,24 @@
+import glob
+import matplotlib.pyplot as plt
+import os
+
+
+fig, axes = plt.subplots()
+pwd = os.path.abspath(os.path.dirname(__file__))
+inv_dir = f"{pwd}/outputs_elev-init-optimization"
+for fpath in glob.glob(f"{inv_dir}_*"):
```

Comment:
If the directory does not exists the scripts exits silently and the used does not necessarily know what happened. So maybe raise an error if glob returns an empty list? Also, as the plot scripts are not tested, you do not need to define `pwd` here? Applies to other plot scripts as well.

Reply:
Good points. Done!